src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchPropertiesSpec.groovy

```diff
@@ -104,6 +105,59 @@ class SwitchPropertiesSpec extends HealthCheckSpecification {
         null                          | "Supported transit encapsulations should not be null or empty"
     }
 
+    @Tidy
+    def "Error is returned when trying to #data.desc"() {
```

Comment:
it is mandatory to explicitly annotate the test with `@Unroll` to get the name pattern unrolled into multiple tests. Otherwise it will look like single test with `#data.desc` not being replaced with proper value

Reply:
fixed